src/test/java/edu/umich/ctools/esb/utils/RunTest.java

```diff
@@ -26,6 +27,20 @@ public Properties getProps() throws IOException{
 		return props;
 	}
 
+//	// TODO: extract out method to pull in grouped properties and place in map.
+//	// should be property file independent
+//	//take input of Props, list of property names, name of group, 
+//	//loop through and return as hashmap
+//			
+//	public static HashMap<String,String> getPropertiesGroup	 (Properties props, String group, List<String> propertyNames) {
+//		HashMap<String, String> value = new HashMap<String, String>();
+//		String key = "tokenServer";
+//		propertyNames.add(key);
+//		
+//		value.put(key, props.getProperty(group + key));
+//		return value;
+//	}
+//	
```

Comment:
I think instead of commenting this out, it should be completely removed. If we need to reference it, this code will remain in history.